src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1601,7 +1664,7 @@ void WholeBodyDynamicsDevice::publishEstimatedQuantities()
             //publishFilteredInertialForGravityCompensator();
 
             //Send filtered force torque sensor measurment, if requested
-            //publishFilteredFTWithoutOffset();
+            publishFilteredFTWithoutOffset();
```

Comment:
you can move the check of `if (streamFilteredFT)` so that there is no unnecessary function jump here.

This could be useful if you're really greedily optimizing the execution time with respect 

"..to remembering the address of the function jump, getting into the function call, doing nothing and coming back to the remembered address." 

this could be avoided.